components/UWAPlantMap.vue

```diff
@@ -1,13 +1,23 @@
 <template>
-  <google-map-loader
-    :map-config="mapConfig"
-    :map-inst.sync="map"
-    :google-inst.sync="google"
-  />
+  <v-card height="100%">
+    <!-- <plant-info :plant-info="plantInfo" /> -->
+    <plant-info
+      :plant-info="plantInfo"
+      :plant-info-visible="plantInfoVisible"
```

Comment:
look into v-model for the close button, there's an example of it that i use for the nav drawer and toolbar